stdlib/public/SDK/ObjectiveC/ObjectiveC.swift

```diff
@@ -196,10 +196,10 @@ typealias Zone = NSZone
 //===----------------------------------------------------------------------===//
 
 @warn_unused_result
-@_silgen_name("objc_autoreleasePoolPush")
+@_silgen_name("_swift_objc_autoreleasePoolPush")
```

Comment:
@gribozavr Do you know if these trampolines are necessary? Is the `ObjectiveC` module far enough long in the build that we could reliably import these from Clang instead?